cmake/FindBlosc.cmake

```diff
@@ -247,7 +249,9 @@ endforeach()
 # Reset library suffix
 
 set(CMAKE_FIND_LIBRARY_SUFFIXES ${_BLOSC_ORIG_CMAKE_FIND_LIBRARY_SUFFIXES})
+set(CMAKE_FIND_LIBRARY_PREFIXES ${_BLOSC_ORIG_CMAKE_FIND_LIBRARY_PREFIXES})
 unset(_BLOSC_ORIG_CMAKE_FIND_LIBRARY_SUFFIXES)
+unset(_BLOSC_ORIG_CMAKE_FIND_LIBRARY_PREFIXES)
 
```

Comment:
What are these changes related to? Do they need a comment in the pending changes?

Reply:
Minor changes to support blosc lib changes in vcpkg